src/hooks/use-loading-messages.ts

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from 'react'
+
+export const LOADING_MESSAGES = [
```

Comment:
😂

Reply:
yah doesn't hurt to add a little whimsy, pretty sure they'll be yanked once we get a better UX flow 😁